journals/apps/journals/models.py

```diff
@@ -43,7 +38,7 @@
 class Organization(models.Model):
     '''Organization Model'''
     name = models.CharField(max_length=255, unique=True)
-    site = models.ForeignKey(
+    site = models.OneToOneField(
```

Comment:
what is the reason for changing this relationship?

Reply:
This will make sure that each organization would be unique site.
This is a good alternative of making `ForeignKey` `unique=True`

Reply:
I think its better to take @bill-filler opinion here.

Reply:
@tuchfarber 
do we want to enforce a 1-1 between Organization and Site?

Reply:
Nope. An organization would be equivalent to a university. For non-WL use cases we would have multiple universities journals connected to a single site.

Reply:
@HammadAhmadWaqas 
So please change this back to how it was. The relationship was correct where a Site can have multiple orgs (for example edX.org (site) could have Georgia Tech (org), Brown (org), etc.)